conda.recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "scikit-rf" %}
-{% set version = "0.21.0" %}
+{% set version = "0.22.0" %}
```

Comment:
This does not seem related. Did you mean to do these things in this PR? There is also another change in `doc/source/conf.py`.